docs/architecture-and-design.md

```diff
@@ -386,6 +390,20 @@ Most [Intents](#intents) are easy to break down into  [Primitives](#primitives),
 
 In order for a robot to move to the desired destination of a `MoveIntent`, the Navigator will use various path-planning algorithms to find a path across the field that does not collide with any robots or violate any restrictions set on the `MoveIntent`. The Navigator then translates this path into a series of `MovePrimitives`, which are sent to the robot sequentially so that it follows the planned path across the field.
 
+### Path Manager
+The `Path Manager` is responsible for generating a set of paths that don't collide. It is given a set of [Path Objective](#path-objective)s and [Path Planner](#path-planner), and it will generate paths using the given path planner and arbitrate between paths to prevent collisions. One simple approach is to add extra obstacles to cause the path planner to generate paths that avoid potential collisions. This approach implicitly uses the idea of [Minkowski space](https://en.wikipedia.org/wiki/Minkowski_space), but where we assume that a robot will occupy all the positions along the path for the next time step.
```

Comment:
The `Minkowski space` isn't mentioned at all in the code. At the very least I think that's very important to include in the in-code documentation (and maybe it should be removed from this doc since it's kinda still related to the implementation).

Reply:
ok moved it